frontend/src/scss/global.scss

```diff
@@ -98,3 +98,7 @@ body {
 .d-flow-root {
   display: flow-root;
 }
+
+.d-contents {
+  display: contents;
```

Comment:
for what do we need this?

Reply:
I need it everytime while prototyping. It is in line with the display helpers of vuetify